src/commands/deploy/deploy.js

```diff
@@ -410,7 +410,10 @@ const bundleEdgeFunctions = async (options) => {
     phase: 'start',
   })
 
-  const { severityCode, success } = await runCoreSteps(['edge_functions_bundling'], { ...options, buffer: true })
+  const { severityCode, success } = await runCoreSteps(['edge_functions_bundling', 'edge_functions_produce_eszip'], {
```

Comment:
I don't think this is doing what you intend. `runCoreSteps` runs a list of core steps, one of which is `edge_functions_bundling`. `edge_functions_produce_eszip` is a feature flag, not a core step.

I think you want to tap into `options.featureFlags`.

Reply:
Oh, how embarrassing. Fixing now.

Reply:
There we go: https://github.com/netlify/cli/pull/4791/commits/6956b93e33229d0cc6c0cdc16edb2e2daf768253 (formatting fixed in subsequent commit).